tilelang/distributed/common/__init__.py

```
@@ -0,0 +1,3 @@
+"""The common distributed modules"""
+
+from .ipc_ext import _create_ipc_handle, _sync_ipc_handles, _create_tensor  # noqa: F401
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)